### PR TITLE
Added cosmetix from the pre-group-refax

### PR DESCRIPTION
**Pull request by ethouris**
Recommended turning off whitespace difs for review.

### CODE REVIEW

File: srtcore/api.cpp [hunk 1]
```diff
@@ -270,15 +270,10 @@ SRTSOCKET CUDTUnited::newSocket()
...
-   // HSv4 there will be only a "socket type" field set
-   // in the handshake, always to UDT_DGRAM.
-   //ns->m_pUDT->m_iSockType = (type == SOCK_STREAM) ? UDT_STREAM : UDT_DGRAM;
-   ns->m_pUDT->m_iSockType = UDT_DGRAM;
    ns->m_pUDT->m_pCache = m_pCache;
 
    // protect the m_Sockets structure.
-   enterCS(m_GlobControlLock);
+   CGuard cs(m_GlobControlLock);
    try
```
Comment (maxsharabayko) [inline srtcore/api.cpp]: What do you think if `CGuard cs(m_GlobControlLock)` is moved inside the try statement?

Comment (ethouris) [reply srtcore/api.cpp]: Well, the only thing guarded by it is the m_Sockets map, and the exception will force the unlock in destructor. Should work.

**Comment by ethouris** [comment]
> * [ ]  Let's extract the renaming of UDP_EPOLL_IN -> SRT_EPOLL_IN etc. into a small PR, so that we get 1 commit in master.

Alright. So hold horses until I make this one.

**Comment by ethouris** [comment]
#1077 is handling the renames.

**Comment by maxsharabayko** [comment]
#1077 is merged - updated this PR

File: srtcore/api.cpp [hunk 2]
```diff
@@ -1507,17 +1526,19 @@ void CUDTUnited::removeSocket(const SRTSOCKET u)
    if (i == m_ClosedSockets.end())
       return;
 
+   CUDTSocket* s = i->second;
+
```
Comment (maxsharabayko) [inline srtcore/api.cpp]: can it be const?

File: srtcore/epoll.cpp [hunk 3]
```diff
@@ -62,13 +62,18 @@ modified by
...
+#include "logging.h"
 
 using namespace std;
 using namespace srt::sync;
 
+#if ENABLE_HEAVY_LOGGING
+static void PrintEpollEvent(ostream& os, int events);
+#endif
+// Use "inline namespace" in C++11
 namespace srt_logging
```
Comment (maxsharabayko) [inline srtcore/epoll.cpp]: No point in this comment. For very distant future?

File: srtcore/handshake.cpp [hunk 4]
```diff
@@ -192,9 +192,12 @@ string CHandShake::show()
     // CHandShake, not CUDT.
     if ( m_iVersion > CUDT::HS_VERSION_UDT4 )
     {
-        so << "EXT: ";
-        if (m_iType == 0) // no flags at all
-            so << "none";
+        int flags = SrtHSRequest::SRT_HSTYPE_HSFLAGS::unwrap(m_iType);
+        so << "FLAGS: ";
```
Comment (maxsharabayko) [inline srtcore/handshake.cpp]: const

File: srtcore/packet.cpp [hunk 5]
```diff
@@ -529,3 +533,71 @@ std::string PacketMessageFlagStr(uint32_t msgno_field)
 
     return out.str();
 }
+
+inline void SprintSpecialWord(std::ostream& os, int32_t val)
+{
```
Comment (maxsharabayko) [inline srtcore/packet.cpp]: SPrint may be better, not to be confused with "sprint"

File: srtcore/packet.h [hunk 6]
```diff
@@ -414,13 +414,13 @@ friend class CRcvQueue;
...
 
-   std::string MessageFlagStr()
 #if ENABLE_LOGGING
-   { return PacketMessageFlagStr(m_nHeader[SRT_PH_MSGNO]); }
+   std::string MessageFlagStr() { return PacketMessageFlagStr(m_nHeader[SRT_PH_MSGNO]); }
+   std::string Info();
 #else
-   { return ""; }
+   std::string MessageFlagStr() { return ""; }
+   std::string Info() { return ""; }
```
Comment (maxsharabayko) [inline srtcore/packet.h]: return `std::string()` to avoid implicit conversions?

File: srtcore/queue.cpp [hunk 7]
```diff
@@ -927,11 +946,14 @@ void CRendezvousQueue::updateConnStatus(EReadStatus rst, EConnectStatus cst, con
         //
         // Maybe the time should be simply checked once and the whole loop not
         // done when "it's not the time"?
-        if (steady_clock::now() >= i->m_tsTTL)
+        steady_clock::time_point now = steady_clock::now();
+        if (now >= i->m_tsTTL)
```
Comment (maxsharabayko) [inline srtcore/queue.cpp]: const

File: srtcore/srt_compat.h [hunk 8]
```diff
@@ -94,7 +94,10 @@ inline struct tm SysLocalTime(time_t tt)
 	if (rr == 0)
 		return tms;
 #else
-	tms = *localtime_r(&tt, &tms);
+
+    // Ignore the error, state that if something
+    // happened, you simply have a pre-cleared tms.
+	localtime_r(&tt, &tms);
```
Comment (maxsharabayko) [inline srtcore/srt_compat.h]: alignment

File: srtcore/core.cpp [hunk 9]
```diff
@@ -8284,12 +8419,35 @@ int CUDT::processData(CUnit *in_unit)
...
+      {
+          int dsize = m_pRcvBuffer->getRcvDataSize();
+          timebufspec << "(" << FormatTime(m_pRcvBuffer->debugGetDeliveryTime(0))
+              << "-" << FormatTime(m_pRcvBuffer->debugGetDeliveryTime(dsize-1)) << ")";
+      }
+
+      std::ostringstream expectspec;
+      if (excessive)
+          expectspec << "EXCESSIVE(" << exc_type << rexmit_reason << ")";
+      else
```
Comment (maxsharabayko) [inline srtcore/core.cpp]: this block looks misaligned

File: srtcore/core.h [hunk 10]
```diff
@@ -283,6 +294,14 @@ class CUDT
     // a different channel.
     void skipIncoming(int32_t seq);
 
+
+    SRTU_PROPERTY_RO(bool, closing, m_bClosing);
+    SRTU_PROPERTY_RO(CRcvBuffer*, rcvBuffer, m_pRcvBuffer);
+    SRTU_PROPERTY_RO(bool, isTLPktDrop, m_bTLPktDrop);
+    SRTU_PROPERTY_RO(bool, isSynReceiving, m_bSynRecving);
```
Comment (maxsharabayko) [inline srtcore/core.h]: To preserve similar naming (see isTLPktDrop, isSynReceiving), better to have `isClosing`

File: srtcore/api.cpp (hunk 2 above)
Comment (ethouris) [reply srtcore/api.cpp]: Depends what exactly. The pointer itself, sure. The designated object (reference) not.

File: srtcore/epoll.cpp (hunk 3 above)
Comment (ethouris) [reply srtcore/epoll.cpp]: Likely. Old times. I'll remove it, it's useless anyway.

File: srtcore/packet.cpp (hunk 5 above)
Comment (ethouris) [reply srtcore/packet.cpp]: The `Sprint` function exists already here. Maybe it's not the happiest name here, but I'd rather postpone the change of the name for this function globally (then both this one and the `Sprint` function would be renamed).

Comment (maxsharabayko) [reply srtcore/packet.cpp]: ok

**Comment by maxsharabayko** [comment]
The build needs to be fixed
```
/home/travis/build/Haivision/srt/srtcore/packet.h:422:32: error: expected primary-expression before ‘return’
    std::string Info() { return return std::string(); }
```

**Pull request merged**